### PR TITLE
LB-215: Remove scheduler from webserver and use cron in a different container.

### PR DESCRIPTION
**Pull request by paramsingh**
This means that scheduled jobs should now be independent
of the webserver. Also fixes docs builds for me in case of
no config file.

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -108,3 +103,15 @@ services:
     depends_on:
       - web
       - rabbitmq
+
+  scheduler:
+    image: web
+    command: cron && tail -f /var/log/cron.log
```
Comment (mayhem) [inline docker/docker-compose.yml]: That's clever! I was wondering how logging was going to work!

File: listenbrainz/bigquery.py [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
 import os
+
 from googleapiclient import discovery
-import googleapiclient
 from oauth2client.client import GoogleCredentials
 
 APP_CREDENTIALS_FILE = os.environ.get('GOOGLE_APPLICATION_CREDENTIALS')
 
+
+logger = logging.getLogger(__name__)
```
Comment (mayhem) [inline listenbrainz/bigquery.py]: You added logging to this module, but it is never used. What's your plan here?

File: listenbrainz/stats/calculate.py [hunk 3]
```diff
@@ -24,12 +27,26 @@ def calculate_user_stats():
...
+cli = click.Group()
+
+@cli.command()
+def calculate():
+    """ Command to calculate statistics from Google BigQuery.
+    This can be used from the manage.py file.
+    """
+
+    if not config.WRITE_TO_BIGQUERY:
+        time.sleep(10000)
```
Comment (mayhem) [inline listenbrainz/stats/calculate.py]: This should be in a loop, so it doesn't try to actually use BQ after 10000 seconds....

File: listenbrainz/bigquery.py (hunk 2 above)
Comment (paramsingh) [reply listenbrainz/bigquery.py]: It was used in this file but I had forgotten to create the logger itself :( https://github.com/paramsingh/listenbrainz-server/blob/a1011edd21a5f330bd6617482baeb91664a39257/listenbrainz/bigquery.py#L18

**Pull request merged**